### PR TITLE
Lesson 1 3

### PR DESCRIPTION
**Pull request by jalvarado9000**

### CODE REVIEW

**Comment by jalvarado9000** [comment]
created new file TodoListItem, moved code from TodoList to TodoListItem and refractor TodoList, update components to use prop, updated AddTodoForm, to not use the default event when submitting the form, saved what the user is typing in order to be showed on the App component.

File: src/AddTodoForm.jsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  
+  const [todoTitle, setTodoTitle] = React.useState('');
+
+ 
+
+  const handleAddTodo  = (event) => {
+    event.preventDefault();
+
+    setTodoTitle(event.target.elements.title.value);
+    console.log(todoTitle);
```
Comment (e-spitz) [inline src/AddTodoForm.jsx]: Make sure to remove any console logs =)

File: src/AddTodoForm.jsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    <div className="text-bg-dark container-lg">
+        <form className="m-5 pb-1" onSubmit={handleAddTodo}>
+          <label htmlFor={todoTitle} className="h5 m-1 mt-3">
+            Title
+          </label>
+          <input name="title" type="text" id={todoTitle} />
+          <input type="submit" className="btn btn-primary m-1 mb-2" value="Add" />
+        </form>
+
+
```
Comment (e-spitz) [inline src/AddTodoForm.jsx]: Check your spacing/formatting here!

File: src/App.jsx [hunk 3]
```diff
@@ -1,27 +1,48 @@
...
-import './App.css'
+import React, { useState } from 'react';
+import './App.css';
+import 'bootstrap/dist/css/bootstrap.css';
+import { AddTodoForm } from './AddTodoForm'
+
+import TodoList from './TodoList';
+import { TodoListItem } from './TodoListItem';
+
+{/* what is the difference between exporting with parenthesis and without it? */ }
```
Comment (e-spitz) [inline src/App.jsx]: Great question! I've added a link in the comments of your latest PR to maybe help guide you with that =) It's all about how you are exporting your component!

File: src/TodoList.jsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+ 
+  return (
+    <div>
+    <h1>Todo List</h1>
+
+    <ul className="list-group">
+        {props.onTodo.map((item) => {
+          return (
+
```
Comment (e-spitz) [inline src/TodoList.jsx]: spacing/indentation here needs some adjusting =)

**Review by e-spitz — CHANGES_REQUESTED** [review]

File: src/App.jsx (hunk 3 above)
Comment (jalvarado9000) [reply src/App.jsx]: I cant find it can you send the link again

**Pull request merged**